assets/react/v3/shared/config/styles.ts

```diff
@@ -418,6 +418,7 @@ export const spacing = {
   8: '8px',
   10: '10px',
   12: '12px',
+  14: '14px',
```

Comment:
This should not be added to our design system. If need to use some spacing out of the design system then use it as hard-coded or use nearby one.

Reply:
Removed!